### DISCUSSION TITLE
Xed already exists

### DISCUSSION THREAD

**Discussion by Markus40** [Xed-Editor/Xed-Editor#20] [General]
Are you aware that there is already a [text editor called xed](https://github.com/linuxmint/xed). How did you come up with that name? Is it just a funny coincidence or did you have the same inspiration? It would be very unlikely to choose the same name by chance.

**Comment by RohitKushvaha01** [comment]
> Are you aware that there is already a [text editor called xed](https://github.com/linuxmint/xed). How did you come up with that name? Is it just a funny coincidence or did you have the same inspiration? It would be very unlikely to choose the same name by chance.

I named the editor "Xed Editor" simply as a random choice; I wasn't aware of the existing text editor by the same name. It's just a coincidence that it matches the name of another editor. 😋
But it was inspired by name of [Zed Editor](https://zed.dev)
